wallet-new/server/Cardano/Wallet/API/V1/Swagger.hs

```diff
@@ -258,14 +257,26 @@ instance ToDocs EstimatedFees where
   descriptionFor _ = "Estimated fees for a `Payment`."
 
 instance ToDocs Payment where
-  descriptionFor _ = "A request for exchange of `Coins` from one entity to another."
+  descriptionFor _ = "An exchange of `Coin`(s) from one source to one or more destinations."
```

Comment:
Minor: word "exchange" seemingly doesn't fit here. Why not use word "transaction"?

Reply:
I have refrained from using "transaction" to not confuse the reader as the input type is `Payment` but we eventually yield a `Transaction`. What about "transfer"? Sounds like a nice middle-ground to me 😉

Reply:
Yes, "transfer" seems good indeed.